data/notices.json

```
@@ -727,6 +727,18 @@
         }
       ],
       "schemaVersion": "1"
+    },
+    {
+      "title": "(cli): may fail on credentials coming from credential provider plugins",
+      "issueNumber": 32653,
+      "overview": "Credential Providers returning empty credentials, or credentials with null expiration fields, would cause the CLI to throw an exception. If you encounter this behavior, upgrade to version 2.173.2 or higher",
+      "components": [
+        {
+          "name": "cli",
+          "version": ">=2.172.0 <= 2.173.1"
```

Not sure it matters but caught my eye. 

```suggestion
          "version": ">=2.172.0 <=2.173.1"
```